internal/cli/auth/login.go

```diff
@@ -172,7 +172,8 @@ func (opts *LoginOpts) oauthFlow(ctx context.Context) error {
 
 	accessToken, _, err := opts.flow.PollToken(ctx, code)
 	var target *atlas.ErrorResponse
-	if errors.As(err, &target) && target.ErrorCode == authExpiredError {
+	tokenExpired := err == auth.ErrTimeout || (errors.As(err, &target) && target.ErrorCode == authExpiredError)
```

Comment:
[non-blocking] feel free to add a function to https://github.com/mongodb/go-client-mongodb-atlas/tree/master/auth, `auth.IsTimeoutErr` or something

Reply:
will do in a separate PR, thx